src/main/antlr4/org/hisp/dhis/parser/expression/antlr/Expression.g4

```diff
@@ -124,6 +124,7 @@ expr
     |   it='d2:relationshipCount(' WS* QUOTED_UID? WS* ')'
     |   it='d2:right(' expr ',' expr ')'
     |   it='d2:round(' expr ')'
+    |   it='d2:round(' expr ',' WS* integerLiteral WS* ')'
```

Comment:
This is OK, though I would have preferred to keep it on one line with a second optional argument as this would be relatively uncomplicated, like the `log(` function.